components/bases-locales/validator/report/index.js

```diff
@@ -82,9 +82,9 @@ Report.propTypes = {
     knownFields: PropTypes.array.isRequired,
     unknownFields: PropTypes.array.isRequired,
     aliasedFields: PropTypes.object.isRequired,
-    rowsWithErrors: PropTypes.array.isRequired,
+    rowsWithIssues: PropTypes.array.isRequired,
     parseMeta: PropTypes.object.isRequired,
-    rowsErrorsCount: PropTypes.number.isRequired,
+    rowsWithIssuesCount: PropTypes.number.isRequired,
```

Comment:
`The prop `report.rowsWithIssuesCount` is marked as required in `Report`, but its value is `undefined``

Reply:
Dans quel cas ?

Reply:
Corrigé